pkg/config/setup/config.go

```diff
@@ -1198,6 +1198,7 @@ func remoteconfig(config pkgconfigmodel.Setup) {
 	config.BindEnvAndSetDefault("remote_configuration.config_root", "")
 	config.BindEnvAndSetDefault("remote_configuration.director_root", "")
 	config.BindEnv("remote_configuration.refresh_interval")
+	config.BindEnv("remote_configuration.org_status_refresh_interval")
```

Comment:
Could you use `config.BindEnvAndSetDefault("remote_configuration.org_status_refresh_interval", 0)` here instead? We're trying to move towards all config options to have defaults so that they have explicit types. If the concern is around the `config.IsSet` method, I have a recommendation for that instead.